### PR TITLE
ivi-input: send surface relative entry cooordinates when pointer focu…

### PR DESCRIPTION
**Pull request by HarshaMM**
…s is forced

Provide surface relative coordinates when pointer focus is forced
to a particular surface instead of providing(0,0). Forcing pointer
focus to a surface is particulary a use case of rotary knobs which
give out pointer axis events only. But applications can end up setting
pointer focus in a simulation environment where the touch input
is unavailable and pointer is used instead.

Signed-off-by: Harsha M M <harsha.manjulamallikarjun@in.bosch.com>

### CODE REVIEW

File: ivi-input-modules/ivi-input-controller/src/ivi-input-controller.c [hunk 1]
```diff
@@ -561,24 +561,21 @@ input_ctrl_ptr_leave_west_focus(struct seat_ctx *ctx_seat,
 
 static void
 input_ctrl_ptr_set_west_focus(struct seat_ctx *ctx_seat,
-        struct weston_pointer *pointer, struct weston_view *w_view,
-        int rel_x, int rel_y)
+        struct weston_pointer *pointer, struct weston_view *view)
```
Comment (eucan) [inline ivi-input-modules/ivi-input-controller/src/ivi-input-controller.c]: Why did you change the name of the parameter from w_view to view in a bugfix patch ? Please do not mix bug-fixing changes and  style changes in one patch.

Comment (HarshaMM) [reply ivi-input-modules/ivi-input-controller/src/ivi-input-controller.c]: ok. I will revert the name changes.

**Comment by HarshaMM** [comment]
@eucan ,
Thanks for the review.
Requested changes are done.
